### PR TITLE
Add sleep ater eating option in russian/

### PR DESCRIPTION
**Pull request by deniszpua**

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: russian/eating-walls/eating-marshmallows.md [hunk 1]
```diff
@@ -4,4 +4,6 @@
 
 Ваши действия:
 
-[Позавтракать с Винипухом и Пяточком](breakfast/breakfast-marshmallows.md)
\ No newline at end of file
+[Позавтракать с Винипухом и Пяточком](breakfast/breakfast-marshmallows.md)
+[Вздремнуть после десерта](fall-asleep/fall-asleep.md)
```
Comment (casey-collab) [inline russian/eating-walls/eating-marshmallows.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**